tests/pivots/pivot_side_panel.test.ts

```diff
@@ -19,6 +19,13 @@ describe("Pivot side panel", () => {
     addPivot(model, "A1:D5", {}, "2");
   });
 
+  test("readonly panel snapshot", async () => {
```

Comment:
IMO snapshot tests are mostly useless, i'ts better to have actual testing of what's in the DOM